content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+Wielu użytkowników Reacta ceni sobie czytanie [Myślenie w Reakcie](/docs/thinking-in-react.html) za możliwość zrozumienia koncepcji. To prawdopodobnie najstarszy przewodnik po Reactcie, jednakże jest on wciąż aktualny.
 
-### Recommended Courses {#recommended-courses}
+### Rekomendowane kursy {#recommended-courses}
 
-Sometimes people find third-party books and video courses more helpful than the official documentation. We maintain [a list of commonly recommended resources](/community/courses.html), some of which are free.
+Czasem ludzie preferują książki i kursy wideo stworzone przez innych użytkowników, uznając je bardziej pomocnymi. Tworzymy [listę rekomendowanych źródeł](/community/courses.html), z których część jest darmowa.
 
-### Advanced Concepts {#advanced-concepts}
+### Zaawansowane koncepty {#advanced-concepts}
 
-Once you're comfortable with the [main concepts](#main-concepts) and played with React a little bit, you might be interested in more advanced topics. This section will introduce you to the powerful, but less commonly used React features like [context](/docs/context.html) and [refs](/docs/refs-and-the-dom.html).
+Kiedy będziesz się już czuł komfortowo z [głównymi konceptami](#main-concepts) i spędził odpowiednio dużo czasu nad pracą w Reakcie, możesz być zainteresowany bardziej zaawansowanymi koncepcjami. Ta sekcja zapozna Cię z przydatnymi, acz rzadziej używanymi cechami jak [context](/docs/context.html) lub [refs](/docs/refs-and-the-dom.html).
 
-### API Reference {#api-reference}
+### Referencje API {#api-reference}
 
-This documentation section is useful when you want to learn more details about a particular React API. For example, [`React.Component` API reference](/docs/react-component.html) can provide you with details on how `setState()` works, and what different lifecycle methods are useful for.
+Ta sekcja dokumentacji jest pożyteczna, jeśli chcesz dowiedzieć się więcej o API Reacta. Na przykład, [`React.Component` referencja API](/docs/react-component.html) pozwoli Ci w szczegółach zrozumieć działanie `setState()`, oraz jak działają metody z cyklu życia komponentu.
 
-### Glossary and FAQ {#glossary-and-faq}
+### Słownik i FAQ {#glossary-and-faq}
 
-The [glossary](/docs/glossary.html) contains an overview of the most common terms you'll see in the React documentation. There is also a FAQ section dedicated to short questions and answers about common topics, including [making AJAX requests](/docs/faq-ajax.html), [component state](/docs/faq-state.html), and [file structure](/docs/faq-structure.html).
+[Słownik](/docs/glossary.html) zawiera przegląd najczęściej używanych terminów, które napotkasz w dokumentacji Reacta. Istnieje także sekcja FAQ, poświęcona krótkim pytaniom i odpowiedziom na najpopularniejsze tematy, między innymi [wykonywanie zapytań AJAX](/docs/faq-ajax.html), [stan komponentu](/docs/faq-state.html), oraz [struktura plików](/docs/faq-structure.html).
```

Comment:
```suggestion
[Słownik](/docs/glossary.html) zawiera przegląd najczęściej używanych terminów, które napotkasz w dokumentacji Reacta. Istnieje także sekcja FAQ, poświęcona krótkim pytaniom i odpowiedziom na najpopularniejsze tematy, między innymi: ["Jak wykonać zapytanie AJAX-owe?"](/docs/faq-ajax.html), ["Co to jest stan komponentu?"](/docs/faq-state.html) oraz ["Jak powinna wyglądać struktura plików?"](/docs/faq-structure.html).
```

Reply:
Tutaj chodzi mi o to, że wstawiłeś nazwy rozdziałów tak po prostu, w środku zdania. Jeśli chcesz to tak przedstawić, to warto dać jakiś dwukropek i przerobić zdanie przed wyliczeniem tak, żeby "przygotowywało" czytelnika do tego, że zaraz coś będziesz wymieniał ;-) Ja bym wstawił jakieś bardziej opisowe linki, czyli pełne pytania, na które podano odpowiedzi w FAQ.